caikit/runtime/names.py

```diff
@@ -248,6 +248,9 @@ def get_task_predict_request_name(
     }
 }
 
+## Service Constants
```

Comment:
Seems to be a duplicate of 5 lines below here?

Reply:
It is! Dang rebase